src/d_iwad.c

```diff
@@ -345,11 +348,198 @@ static void CheckInstallRootPaths(void)
 
 // Check for Doom downloaded via Steam
 
+#define MAX_STEAM_LIBRARIES 16
+
+static char *steam_libraries[MAX_STEAM_LIBRARIES];
+static int num_steam_libraries = 0;
+
+static void AddSteamLibraryDir(char *dir)
```

Comment:
Do you need the `steam_libraries[]` array? Can't you just make this a wrapper around AddIWADDir?

Reply:
Yes, that array is note needed.